extensions/wikia/RWEPageHeader/scripts/rwe-page-header.js

```diff
@@ -1,4 +1,4 @@
-require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracker) {
+require(['wikia.window', 'jquery', 'wikia.tracker', 'wikia.onScroll'], function (window, $, tracker, onScroll) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "onScroll". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)